experimental/tools/elastic-miter/CMakeLists.txt

```diff
@@ -14,14 +14,10 @@ add_llvm_tool(elastic-miter
 llvm_update_compile_flags(elastic-miter)
 target_link_libraries(elastic-miter
   PRIVATE
-  AffineToScf
-  DynamaticLowerScfToCf
   DynamaticBufferPlacement
   DynamaticLSQSizing
   DynamaticExperimentalResourceSharing
-  DynamaticCfToHandshake
   DynamaticHandshake
-  DynamaticHandshakeToHW
   DynamaticTestTransforms
   DynamaticTransforms
   DynamaticTutorialsCreatingPasses
```

Comment:
Are these really needed to be linked? This makes the compilation of other parts of Dynamatic slow (I experienced it since the merge of ElasticMiter PR on my laptop), since they need to be linked to elastic-miter
I removed apparently unnecessary ones and I'm still not sure about the others